### PR TITLE
Configurable ingress proxy settings

### PR DESCRIPTION
**Pull request by T0bl**
## Pull request description

The default value doesn't fit if i'am running the IntegrityChecker in a large cluster -> Proxy Timeout

## Checklist
- [x] I have added unit tests
- [x] I have applied the change to all applicable products
- [x] I have added the change description to the `changelog.md` and `Chart.yaml` files
- [x] (Atlassian only) [Internal Bamboo CI is passing](https://server-syd-bamboo.internal.atlassian.com/browse/DCD-K8SHELMTEST71-1)

### CODE REVIEW

**Comment by nanux** [comment]
@T0bl, thank you for the contribution! What was the main reason why the static values didn't suit? Could you please add some basic use cases to the PR description?

Do you think you could also add the same options to the other products (the implementation will be basically the same)? We are striving for feature parity between the products and this change will be useful in all of them.

It would be very useful to add test for these specific values to https://github.com/atlassian/data-center-helm-charts/blob/e79645670be0948af188a43ec74d47315e4dfd24/src/test/java/test/IngressTest.java

Thank you

**Comment by errcode1202** [comment]
> 

To add to this @T0bl it would be great if you could add a unit test(s) to [IngressTest.java](https://github.com/atlassian/data-center-helm-charts/blob/main/src/test/java/test/IngressTest.java) ensuring that these new properties you're defining in `values.yaml` are ultimately set against the ingress itself.

**Comment by T0bl** [comment]
Hi @nanux,  i've added the options also for the other products and updated the description.

**Comment by errcode1202** [comment]
> fixed the ingress checks

Cheers @T0bl. We'll get this reviewed today.

File: src/main/charts/bitbucket/Changelog.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+## 1.4.0
+
+**Release date:** TBD
+
+![AppVersion: 7.13.5](https://img.shields.io/static/v1?label=AppVersion&message=7.13.5&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+* Support for configuring ingress proxy settings via values.yaml
```
Comment (nghazali) [inline src/main/charts/bitbucket/Changelog.md:11]:
```suggestion
* Support for configuring ingress proxy settings via values.yaml (#402)
```

Comment (nghazali) [reply src/main/charts/bitbucket/Changelog.md:11]: Would you please add the PR# to `Changelog.md` for other products?

**Review by errcode1202 — APPROVED** [review]

**Review by nghazali — APPROVED** [review]

**Review by nanux — APPROVED** [review]

**Review by jjeongatl — APPROVED** [review]

**Pull request merged**